lib/extensions.js

```diff
@@ -41,6 +41,7 @@ function getHumanArchitecture(arch) {
     case 'ia32': return '32-bit';
     case 'x86': return '32-bit';
     case 'x64': return '64-bit';
+    case 'ppc64': return '64-bit';
```

Comment:
2 cents, I think the human readable version might be "PowerPC 64-bit" rather than normalizing it like the rest to just "64-bit"